LibreNMS/Alert/Transport/Telegram.php

```diff
@@ -79,11 +84,23 @@ public static function configTemplate()
                     'name' => 'telegram-token',
                     'descr' => 'Telegram Token',
                     'type' => 'text',
+                ],
+                [
+                    'title' => 'Format',
+                    'name' => 'telegram-format',
+                    'descr' => 'Telegram format',
+                    'type' => 'select',
+                    'options' => [
+                        'Markdown' => 'Markdown',
```

Comment:
I guess you can also add a blank option as well with no value.

Reply:
Correct. My mistake. Added now.